ddl/ddl_api.go

```diff
@@ -684,12 +684,13 @@ func (d *ddl) CreateTable(ctx context.Context, ident ast.Ident, colDefs []*ast.C
 	return errors.Trace(err)
 }
 
-// If create table with auto_increment option, we should rebase tableAutoIncID value.
+// handleAutoIncID increases AutoIncID by one. Rebase table's AutoIncID value, when table created with auto_increment option
```

Comment:
handleAutoIncID handles auto_increment option in DDL. It creates a ID counter for the table and initiates the counter to a proper value. 
For example if the option sets auto_increment to 10. The counter will be set to 9. So the next allocated ID will be 10.